script/Ciel/Day 5/05_01_CIEL05_1C.txt

```diff
@@ -910,7 +910,7 @@ Her teeth sink into the joint right behind my fingertip.
 [sha:58086bf47add25f107175c2eb504db96923da08f]{
 -- Page 145, Offset 11275. Mods: @n.
 -- 「……！
-"...!
+"!
 }
 [offset:11276]{
 -- Page 145, Offset 11276. Glued. Mods: @x, @n.
```

Comment:
This is Noel struggling with Shiki's hand in her mouth, as above.
Should probably use italics?

11276
>Mrgh—mrm

11278
>mmhhrr

11280
>Chomp!

11281
Just don't think three exclamation-marks are necessary.
>!